keyboards/doro67/multi/keymaps/konstantin/rules.mk

```diff
@@ -1,5 +1,5 @@
 # Generic features
-BOOTMAGIC_ENABLE = lite     # Enable Bootmagic Lite
+BOOTMAGIC_ENABLE   = lite
```

Comment:
```suggestion
BOOTMAGIC_ENABLE   = yes
```

Reply:
Oh, nice, we got there already! I really like the pace at which QMK development (and, specifically, refactoring and cleanup) has been happening as of late :)